doc/DECRYPTION_EXAMPLE.md

```diff
@@ -0,0 +1,123 @@
+
+- Open your browser and navigate to:
+
+#### [https://sandbox.bcda.cms.gov/api/v1/swagger](https://sandbox.bcda.cms.gov/api/v1/swagger/)
+<img src="images/decrypt_demo_01.png" alt="Screenshot of swagger" width="400" />
+
+Let's provide our token to get full access to the API.
+
+<img src="images/decrypt_demo_02.png" alt="Authorization dialog" width="350" />
+
+- Click the `Authorize` button
+- Paste your `token` in the `api_key` box
+- Click `Authorize`, then `Close`
+
+<img src="images/decrypt_demo_03.png" alt="API menu" width="600" />
+
+We have two types of encrypted files we could download: Patient and Explanation of Benefit (EoB).  Let's choose EoB today.
+
+- Click the `/api/v1/ExplanationOfBenefit/$export` link
+
+Notice that there's quite a bit of documentation here, from the parameter types to possible kinds of server responses.
+
+<img src="images/decrypt_demo_04.png" alt="Starting Explanation of Benefits job" width="600" />
+
+- Click `Try it out`, then `Execute`
+
+<img src="images/decrypt_demo_05.png" alt="Explanation of Benefits response" width="600" />
+
+If you'd like to repeat this from the command line or implement this API call in code, look in the `Curl` section for the request we just made.  Not far below that under `Server response` you can see the response: an `HTTP` 202 success giving a link in the `content-location` header for status information on our EoB job.  
+
+- Note the job ID number at the end of this link.
+- Open the job status section in Swagger (click `/api/v1/jobs/{jobID}`)
+
+<img src="images/decrypt_demo_06.png" alt="Requesting job status" width="600" />
+
+- Type the job ID we received
+- Click `Execute`
+
+<img src="images/decrypt_demo_07.png" alt="Job status response" width="600" />
+
+Depending on the size of the file, the job may take some time.  If the job is not yet complete, this will give us status information.  Shown below are the results for our completed job.  We can download the file from the URL provided.
+
+Take special note of the new `KeyMap` section of the response.  To decrypt the file, we will need the filename (the first part of the keymap) and the [symmetric key](../ENCRYPTION.md#how-we-encrypt) (the second part of the keymap), as shown above.  There are no spaces in either one.
+
+- Copy these values (filename and symmetric key) for later.
+
+Our last API task is to download the encrypted file. 
+
+- Open the data file section in Swagger (click `/api/v1/jobs/{jobID}/{filename}`)
+- Paste the job ID and filename into the appropriate boxes
+- Click `Execute`
+
+<img src="images/decrypt_demo_08.png" alt="Get data file (Swagger)" width="600" />
+
+- Click the `Download file` link that appeared in the response section.  Note that a large sample file may take a while to download.
+
+<img src="images/decrypt_demo_09.png" alt="Download file" width="600" />
```

Comment:
Does this image contain a token?   Should we obfuscate it a bit?

Reply:
Yes, a local token that's never been near one of our servers.  I don't think obfuscation improves our security, and I think it detracts somewhat from readability.